src/vendor/studio.scss

```diff
@@ -1378,15 +1342,15 @@ main {
     }
 
     h3 {
-      @extend %t-title7;
+      @extend .h5;
```

Comment:
You used extend `.h5` class for `h3` tag, it is correct?

Reply:
Yes, because this tag looks like h5 title